### PR TITLE
Removed Duplicated instructions from READMEs

### PR DESCRIPTION
**Pull request by caleonardo**

### CODE REVIEW

File: 1-org/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
 1. Navigate into the repo `cd gcp-org` and change to a non production branch `git checkout -b plan`
 1. Copy contents of foundation to new repo `cp -RT ../terraform-example-foundation/1-org/ .` (modify accordingly based on your current directory).
 1. Copy cloud build configuration files for terraform `cp ../terraform-example-foundation/build/cloudbuild-tf-* . ` (modify accordingly based on your current directory).
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (rjerrems) [inline 1-org/README.md:23]: It looks like the second part is still duplicated :)

File: 1-org/README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
     _TF_SA_EMAIL
     _STATE_BUCKET_NAME
     ```
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```
Comment (rjerrems) [inline 1-org/README.md:43]: same as above

File: 1-org/README.md (hunk 1 above)
Comment (caleonardo) [reply 1-org/README.md]: Can't believe it. Thank you Rohan!

File: 1-org/README.md (hunk 2 above)
Comment (caleonardo) [reply 1-org/README.md]: Done! thank you

**Pull request merged**